services/continuousphp/continuousphp.service.js

```diff
@@ -17,15 +17,26 @@ module.exports = class ContinuousPhp extends LegacyService {
   static get route() {
     return {
       base: 'continuousphp',
-      pattern: '',
+      pattern: ':provider/:user/:repo/:branch*',
     }
   }
 
   static get examples() {
     return [
       {
         title: 'continuousphp',
-        previewUrl: 'git-hub/doctrine/dbal/master',
+        pattern: ':provider/:user/:repo/:branch',
+        namedParams: {
+          provider: 'git-hub',
```

Comment:
Sidenote, I've never seen GitHub split like this before

Reply:
Yea, it's really weird.